install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -369,6 +366,8 @@ spec:
           mountPath: /var/lib/cilium/tls/hubble
           readOnly: true
         {{- end }}
+        - name: tmp
+          mountPath: /tmp
```

Comment:
is `/tmp` the best path? It seems very generic to me, if it's all the same why not something like `/etc/cilium`?